pkgdb/include/flox/flox-flake.hh

```diff
@@ -13,6 +13,7 @@
 #include <memory>
 #include <nix/eval.hh>
 #include <nix/flake/flake.hh>
+#include <nix/flake/flakeref.hh>
```

Comment:
> Add wrappers around `lockFlake` and `getFlake` to ensure downloads of flakes occur in threads.

Is `threads` in the PR description a typo?

Reply:
Sure is! Thanks.